common/reviews/api/tools.api.md

```diff
@@ -1117,11 +1117,51 @@ declare namespace cursors {
         elementCursor,
         registerCursor,
         CursorNames,
+        CursorSVG,
         setCursorForElement
     }
 }
 export { cursors }
 
+// @public (undocumented)
+const CursorSVG: {
+    Angle: SVGCursorDescriptor;
+    ArrowAnnotate: SVGCursorDescriptor;
+    Bidirectional: SVGCursorDescriptor;
+    CobbAngle: SVGCursorDescriptor;
+    CircleROI: SVGCursorDescriptor;
+    EllipticalROI: SVGCursorDescriptor;
+    FreehandROI: SVGCursorDescriptor;
+    FreehandROISculptor: SVGCursorDescriptor;
+    Length: SVGCursorDescriptor;
+    Probe: SVGCursorDescriptor;
+    RectangleROI: SVGCursorDescriptor;
+    TextMarker: SVGCursorDescriptor;
+    Crosshairs: SVGCursorDescriptor;
+    Eraser: SVGCursorDescriptor;
+    Magnify: SVGCursorDescriptor;
+    Pan: SVGCursorDescriptor;
+    Rotate: SVGCursorDescriptor;
+    StackScroll: SVGCursorDescriptor;
+    WindowLevelRegion: SVGCursorDescriptor;
+    WindowLevel: SVGCursorDescriptor;
+    Zoom: SVGCursorDescriptor;
+    segmentationFreeHandEraseInside: SVGCursorDescriptor;

```

Comment:
These should match the casing of all the others. What is segmentationFreeHand anyway?